pkg/segment/segment.go

```diff
@@ -105,29 +108,124 @@ func (s *segmentStore) Put(ctx context.Context, path paths.Path, data io.Reader,
 	}
 
 	// puts pointer to pointerDB
-	_, err = s.pc.Put(ctx, &pr)
+	_, err = s.pdb.Put(ctx, &pr)
 	if err != nil || status.Code(err) == codes.Internal {
 		zap.L().Error("failed to put", zap.Error(err))
 		return Error.Wrap(err)
 	}
 	return nil
 }
 
-// Get retrieves a file from the erasure code client with help from overlay and pointerdb
+// Get retrieves a file using erasure code, overlay, and pointerdb clients
 func (s *segmentStore) Get(ctx context.Context, path paths.Path) (ranger.Ranger, Meta, error) {
-	m := Meta{
-		Data: []byte(""),
+	m := Meta{}
+	// TODO: remove this chunk after pointerdb client interface merged
+	gr := &ppb.GetRequest{
+		Path:   []byte(fmt.Sprintf("%s", path)),
```

Comment:
nit: `fmt.Sprintf("%s", thing)` just calls `thing.String()`. so we can avoid fmt.Sprintf entirely?